CMakeLists.txt

```diff
@@ -82,12 +83,10 @@ if(NOT FMOD_DIR)
   message(STATUS "FMOD_DIR not set. No audio.")
 else()
   message(STATUS "FMOD_DIR: ${FMOD_DIR}")
+  add_definitions(-DHAVE_FMOD_LIBRARY)
   if(WIN32)
     target_link_libraries(${CMAKE_PROJECT_NAME} ${FMOD_DIR}/api/lowlevel/lib/fmod_vc.lib)
     target_link_libraries(${CMAKE_PROJECT_NAME} ${FMOD_DIR}/api/studio/lib/fmodstudio_vc.lib)
-  else()
```

Comment:
Are we sure this works for Mac/Linux? I thought that's what the else was for

Reply:
It doesn't work for Max or Linux. I got rid of the else because I don't know what libraries Mac or Linux would have to use. I don't have a Mac or Linux OS, so I can't really improve on this until Brennan or Jacob can tell me what those libraries are.